### PR TITLE
RST Conversion

### PR DESCRIPTION
**Pull request by gonsie**
All specs have been converted to RST files. 

The README has been converted to markdown, so that navigating to a link on GitHub will work (to support current use case of browsing on GitHub).

### CODE REVIEW

**Comment by grondo** [comment]
Woo-hoo, thanks!

File: spec_15.rst [hunk 1]
```diff
@@ -277,71 +330,88 @@ ownership by ensuring that the current container includes the
...
-   without authority. The _intended recipient_ field of the user request
+   without authority. The *intended recipient* field of the user request
    protects against users other than the instance owner using the
    guest request, and a fixed time-to-live prevents the request from
-   being used indefinitely. Finally, the `flux-security-imp` logs all
+   being used indefinitely. Finally, the ``flux-security-imp`` logs all
    invocations, thereby allowing replays to be detected and audited.
 
-[footnotes]
+.. [1]
```
[the remark this answers is not in the document]
Comment (gonsie) [reply spec_15.rst]: This should be fixed. It does show as a weird table thing... but it should all be in proper sphinx formatting.

File: spec_4.rst [hunk 2]
```diff
@@ -281,69 +330,79 @@ The implementors of the Flux Resource Model MAY
...
 
+Walker
+   Provide generic ways to traverse the graph, visiting
+   a subset of its vertices with a specific traversal pattern.
+   It is passed in the starting vertex and the name of a subsystem
+   (e.g., the root vertex of a compute-hardware containment hierarchy
+   or an I/O bandwidth hierarchy) from which to walk.
+   In particular, on a tree hierarchy, preorder and postorder
+   visiting patterns SHALL be supported, and a user MAY be able to
+   register with it pre- and/or post-order callbacks, or ``filters'',
```
[the remark this answers is not in the document]
Comment (gonsie) [reply spec_4.rst]: fixed for all double quote literals from adoc. thanks!

**Comment by gonsie** [comment]
@SteVwonder I'd appreciate another go-through, thanks!

File: Makefile [hunk 3]
```diff
@@ -15,6 +15,8 @@ SCHEMA_DIRS=data/spec_26 data/spec_14
 help:
 	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
+.PHONY: help Makefile check $(SCHEMA_DIRS)
```
Comment (SteVwonder) [inline Makefile]: Really minor nit:
```suggestion
.PHONY: help Makefile check spelling $(SCHEMA_DIRS)
```

**Comment by SteVwonder** [comment]
LGTM! If you want to squash your commits, I’ll add the merging label afterwards.

**Comment by gonsie** [comment]
Let me know if I mucked it up completely 😕 but I think this is ready.

**Comment by SteVwonder** [comment]
It looks like a merge commit slipped in at the end.  Speaking with @grondo, if you do `git rebase -i upstream/master` (or whatever your pointer to the upstream remote is) and then drop that last merge commit, it should be good to go after that.

**Comment by gonsie** [comment]
yeahhhhhh I clicked the dumb button in GitHub. For really realz this time!

**Comment by SteVwonder** [comment]
Thanks @gonsie!!!!

**Pull request merged**